packages/astro/src/core/build/pipeline.ts

```diff
@@ -38,9 +38,12 @@ export class BuildPipeline extends Pipeline {
 	#routesByFilePath: WeakMap<RouteData, string> = new WeakMap<RouteData, string>();
 
 	get outFolder() {
-		return this.settings.buildOutput === 'server'
-			? this.settings.config.build.server
-			: getOutDirWithinCwd(this.settings.config.outDir);
+		const out =
```

Comment:
just noting that eventually this will need to be removed, looks like this is used to dynamically import an entrypoint. There aren't multiple entrypoints any more. So maybe a comment would be good here?